Makefile

```diff
@@ -111,7 +111,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 	$(CONTROLLER_GEN) \
 		rbac:roleName=manager-role output:rbac:artifacts:config=config/components/rbac\
 		webhook output:webhook:artifacts:config=config/components/webhook\
-		paths="./pkg/controller/...;./pkg/webhooks/...;./pkg/util/cert/..."
+		paths="./pkg/controller/...;./pkg/webhooks/...;./pkg/util/cert/...;./pkg/visibility/..."
```

Comment:
This seems to be breaking the build. I remember there was an issue with including the `/pkg/visibility` from scanning. Maybe we could move the markers to the `/apis/visibility`, or scan just a sub-package of `pkg/visibility` which contains the markers.

Reply:
I'll put these markers on `/pkg/visibility/api`. That should work fine.

Reply:
nice, thanks for investigating